helpers/search_processing_kendra_quickstart.sh

```diff
@@ -0,0 +1,228 @@
+fi
+if [ -z "${OPENSEARCH_DASHBOARDS_IMAGE_TAG:-}" ]; then
+  OPENSEARCH_IMAGE_TAG="opensearchproject/opensearch-dashboards:${OPENSEARCH_VERSION}"
+fi
+if [ -z "${SEARCH_PROCESSOR_PLUGIN_URL:-}" ]; then
+  SEARCH_PROCESSOR_PLUGIN_URL="https://github.com/opensearch-project/search-relevance/releases/download/${OPENSEARCH_VERSION}/search-processor.zip"
+fi
+
+
+function print_help() {
+  echo "Usage: $0 -p <execution_plan_id> [-r <region>] [-e <kendra_ranking_endpoint>]"
+  echo "  -p | --execution-plan-id          The ID returned from Kendra Intelligent Ranking service"
+  echo "                                    from the call to CreateRescoreExecutionPlan"
+  echo "  -r | --region                     The AWS region for the Kendra Intelligent Ranking"
+  echo "                                    service endpoint. If not specified, will read from the"
+  echo "                                    AWS CLI for the default profile."
+  echo "  -e | --kendra-ranking-endpoint    The URL for the Kendra Intelligent Ranking service"
+  echo "                                    endpoint. If not specified, a value based on the"
+  echo "                                    region will be used."
+  echo
+  echo "  NOTE: The script will attempt to read AWS credentials (access/secret key)"
+  echo "  from environment variables, and then from the default AWS profile, in order to"
+  echo "  pass them to the OpenSearch keystore to be used to connect to the Kendra Intelligent"
+  echo "  ranking service."
+  echo "  If no credentials are found, the script WILL NOT pass credentials to the OpenSearch"
+  echo "  keystore. When running a reranking request, the ranking plugin may rely on"
+  echo "  instance profile credentials delivered through the EC2 metadata service."
+}
+
+#
+# Parse and validate arguments
+#
+
+while [ "$#" -gt 0 ]; do
+    case $1 in
+        -p | --execution-plan-id )
+            shift
+            EXECUTION_PLAN_ID=$1
+            shift
+            ;;
+        -r | --region )
+            shift
+            AWS_REGION=$1
+            shift
+            ;;
+        -e | --kendra-ranking-endpoint )
+            shift
+            KENDRA_RANKING_ENDPOINT=$1
+            shift
+            ;;
+        -h | --help )
+	    print_help
+            exit 0
+            ;;
+    esac
+done
+
+ERROR=0
+if [ -z "${EXECUTION_PLAN_ID:-}" ]; then
+  >&2 echo "Missing argument [-p | --execution-plan]"
+  ERROR=1
+fi
+if [ -z "${AWS_REGION:-}" ]; then
+  AWS_REGION=$(aws configure get region)
+  if [ -z "${AWS_REGION:-}" ]; then 
+    >&2 echo "Missing argument [-r | --region] and could not infer from 'aws configure get region'"
+    ERROR=1
+  fi
+fi
+
+# End of required arguments. If any are missing, print help and exit.
+if [ "${ERROR}" == "1" ]; then
+  echo
+  print_help
+  exit 1
+fi
+
+if [ -z "${KENDRA_RANKING_ENDPOINT:-}" ]; then
+  KENDRA_RANKING_ENDPOINT="https://kendra-ranking.${AWS_REGION}.amazonaws.com"
```

Comment:
We are actually changing the default endpoint to follow the format `https://kendra-ranking.${AWS_REGION}.api.aws`. Can you please update this? Thanks!

Reply:
Thanks, fixed!